src/Analyser/MutatingScope.php

```diff
@@ -2471,7 +2472,7 @@ static function (Node $node, Scope $scope) use ($arrowScope, &$arrowFunctionImpu
 
 					return $cloneType;
 				}
-				$resolvedType = $this->getDynamicFunctionReturnType($parametersAcceptor, $normalizedNode, $functionReflection);
+				$resolvedType = $this->getDynamicFunctionReturnType($normalizedNode, $functionReflection);
```

Comment:
in this path $node was already normalized in line 2449 and before this PR will get normalized again within `getDynamicFunctionReturnType`